src/test/resources/properties/defaultTest.properties

```diff
@@ -0,0 +1,26 @@
+server.listenAddress=127.0.0.1
+server.listenPort=8080
+server.threads=1
```

Comment:
create a multithreaded test as well and use 10 clients, 100 messages each in paralel and check results.

Reply:
New test added in MultithreadingEndToEndTest.java